rpcd/playbooks/ansible.cfg

```diff
@@ -12,19 +12,19 @@ timeout = 120
 
 # Set the path to the folder in os-ansible-deployment which holds the dynamic
 # inventory script - new config setting for ansible v1.9 and above
-inventory = /opt/os-ansible-deployment/playbooks/inventory/
+inventory = ../../os-ansible-deployment/playbooks/inventory/
```

Comment:
Why are we using ../.. here and /opt everywhere else? Rather use one convention and stick to it. I would recommend rather replacing '../..' with '/opt'